README.md

```diff
@@ -57,34 +57,93 @@ And then execute:
 bundle install
 ```
 
-## Usage
 
-Once orchid is installed successfully, run a generator to prepare your new rails app. Run this with a `--help` to see what will be changed before you begin. This will NOT erase or overwrite any configuration files, only example configuration files.
+## Usage
+Once Orchid is installed successfully, run the generator to prepare your new rails app. Run this with a `--help` to see which app files will be changed before you begin.
```

Comment:
I realize this was in before your PR, but the `--help` option suggests some arguments that we aren't respecting:

```
Runtime options:
  -f, [--force]                    # Overwrite files that already exist
  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
  -q, [--quiet], [--no-quiet]      # Suppress status output
  -s, [--skip], [--no-skip]        # Skip files that already exist
```